docs/source/glossary.md

```diff
@@ -124,4 +124,13 @@ Hierarchical Ordinary Differential Equation
 [MCMC](https://en.wikipedia.org/wiki/Markov_chain_Monte_Carlo)
   Markov chain Monte Carlo (MCMC) methods comprise a class of algorithms for sampling from a probability distribution. By constructing a {term}`Markov Chain` that has the desired distribution as its equilibrium distribution, one can obtain a sample of the desired distribution by recording states from the chain.  Various algorithms exist for constructing chains, including the Metropolis–Hastings algorithm.
 
+tensor_like
+  Any scalar or sequence that can be interpreted as a TensorVariable. In addition to TensorVariables, this list includes NumPy ndarrays, scalars, lists and tuples (possibly nested). Any argument accepted by aesara.tensor.as_tensor_variable is tensor_like.
```

Comment:
```suggestion
  Any scalar or sequence that can be interpreted as a {class}`~aesara.tensor.TensorVariable`. In addition to TensorVariables, this list includes NumPy ndarrays, scalars, lists and tuples (possibly nested). Any argument accepted by aesara.tensor.as_tensor_variable is tensor_like.
```

completely overlooked that. this should be a clickable link in its first occurrence, but numpydoc aliases only work for types in docstrings, not in text.

Reply:
Added